post_installation_configuration/configuring-multi-arch-compute-machines/multiarch-tuning-operator.adoc

```diff
@@ -12,21 +12,29 @@ The Multiarch Tuning Operator is a Technology Preview feature only. Technology P
 For more information about the support scope of Red{nbsp}Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
 ====
 
-The Multiarch Tuning Operator enhances the operational experience within multi-architecture clusters, and single-architecture clusters that are migrating to a multi-architecture compute configuration. 
+The Multiarch Tuning Operator supports architecture-aware workload scheduling on multi-architecture clusters, and single-architecture clusters that are migrating to a multi-architecture compute configuration.
 
-This Operator implements the `ClusterPodPlacementConfig` object to support architecture-aware workload scheduling.
```

Comment:
I think we might not want to discuss a lot of design info. 

In the intro section, I have added this info:
"To enable architecture-aware workload scheduling, you must create the ClusterPodPlacementConfig object by using the Multiarch Tuning Operator. When you create the ClusterPodPlacementConfig object, this Operator deploys an operand." 

I thought this would provide a clear flow of events. Like the users would know when and how the Operand is created by the Operator.